include/tue/config/reader.h

```diff
@@ -22,14 +22,14 @@ class Reader
 
     virtual ~Reader();
 
-    bool read(const std::string& name);
+    bool read(const std::string& name, const NodeType type, const RequiredOrOptional opt = OPTIONAL);
```

Comment:
Does this affect other packages than ED?

Reply:
This for internal use. This creates a difference for map/array. readGroup/readArray always returned true, as long as a child with that name was available, even if it was the wrong type.

Reply:
Function is now private.